aiohttp_devtools/runserver/serve.py

```diff
@@ -59,17 +59,19 @@ def get_host(request):
         async def static_middleware(app, handler):
 
             async def _handler(request):
-                static_url = 'http://{}:{}/{}'.format(get_host(request), config.aux_port, static_path)
-                dft_logger.debug('settings app static_root_url to "%s"', static_url)
-                app['static_root_url'] = static_url
+                if config.static_path is not None:
```

Comment:
can you remove `if config.static_path is not None` here and change above to 

```py
if config.infer_host and config.static_path is not None:
```

We should also change this to use the new version of aiohttp middleware, you can either do this here or I'll do another PR

Reply:
Maybe, this middleware can be left out entirely?
It changes app['static_root_url'] on evey request, but indeed, it was already changed in the body of modify_main_app(), that is executed (at least i guess so) when setting up the app.

Also, I'm not very deep in the details of aiohttp_devtools, so maybe there is a reason to have the middleware, too. But with the quick glance I have taken, I couldn't see a reason.